modules/command-not-found/init.zsh

```diff
@@ -11,6 +12,9 @@ if [[ -s '/etc/zsh_command_not_found' ]]; then
 # Load command-not-found on Arch Linux-based distributions.
 elif [[ -s '/usr/share/doc/pkgfile/command-not-found.zsh' ]]; then
   source '/usr/share/doc/pkgfile/command-not-found.zsh'
+# Load command-not-found on Mac OS X with https://github.com/bfontaine/homebrew-command-not-found installed
+elif [[ $(type -f brew) ]] && [[ -s ${HANDLER=$(brew --prefix)/Library/Taps/bfontaine/homebrew-command-not-found/handler.sh} ]]; then
```

Comment:
You can use `(( $+commands[brew] ))` to check for `brew`.
Don't define variables inside of `if` statement checks.